app/assets/stylesheets/base/buttons.scss

```diff
@@ -1,7 +1,7 @@
 $btn-radius: 5px;
 
 .btn {
-  background: $primary;
+  background: $secondary;
```

Comment:
@thatdevguy1 looking at this change, im thinking that we should extract the bgcolor out of the .btn class and then have a subclass defined for the colours.

so we would use it : btn (base style) btn-whatever (add on styles)